src/dcos_e2e/node.py

```diff
@@ -31,13 +31,15 @@ def __init__(self, ip_address: IPv4Address, ssh_key_path: Path) -> None:
     def compose_ssh_command(
         self,
         args: List[str],
+        user: str,
         env: Optional[Dict]=None,
     ) -> List[str]:
         """
         Run the specified command on the given host using SSH.
 
         Args:
             args: The command to run on the node.
+            user: The username to SSH.
```

Comment:
This is ambiguous - it maybe says that `user` will perform the SSH command. Make it clear that this is the username for a user on the node which will run the command.